symengine/tests/polynomial/test_uratpoly_flint.cpp

```diff
@@ -3,7 +3,9 @@
 
 #include <symengine/polys/uintpoly_flint.h>
 #include <symengine/pow.h>
+#include <symengine/symengine_exception.h>
```

Comment:
You don't need this include everywhere because it is included in `basic.h` and `basic.h` is included in almost all files.